Formula/python@3.9.rb

```diff
@@ -343,12 +382,13 @@ def caveats
   end
 
   test do
-    xy = (prefix/"Frameworks/Python.framework/Versions").children.min.basename.to_s
     # Check if sqlite is ok, because we build with --enable-loadable-sqlite-extensions
     # and it can occur that building sqlite silently fails if OSX's sqlite is used.
     system "#{bin}/python#{xy}", "-c", "import sqlite3"
```

Comment:
```suggestion
     system "#{bin}/python#{version.major_minor}", "-c", "import sqlite3"
```

Reply:
`xy` returns the version found in `Python.framework/Versions`